pending.py

```diff
@@ -68,13 +68,17 @@ def iterate_prs(repo, skip, users, path):
             print_color('red', "SKIP untrusted user")
         elif (user, branch) in skip:
             print_color('yellow', "SKIP")
-        elif pr.base.ref != args.main:
-            print_color('yellow', "SKIP base: {}".format(pr.base.ref))
+        elif (pr.base.ref != args.main and
+              (pr.base.ref not in include_spec['search_branches'] and
```

Comment:
What's the point of "search branches" when we already have `args.main`?  I think a branch + a label that matches the branch would be fine.

Reply:
As I mentioned above, its possibly because I wanted to create some kind of a generic mechanism which is actually not needed. Let me rework that.